File: src/test/java/org/code4everything/hutool/HutoolAlias.java
package org.code4everything.hutool;

import cn.hutool.core.io.FileUtil;
import com.alibaba.fastjson.JSON;

/**
 * @author pantao
 * @since 2021/5/11
 */
public class HutoolAlias {

    public static void main(String[] args) {
        String json = FileUtil.readUtf8String(Hutool.workDir + "\\class.json");
        JSON.parseObject(json).forEach((k, v) -> {
            String arg = k + "#alias -d";
            Hutool.main(arg.split(" "));
            System.out.println();
        });
    }
}

File: src/test/kotlin/org/code4everything/hutool/HutoolAlias.kt
package org.code4everything.hutool

import cn.hutool.core.io.FileUtil
import com.alibaba.fastjson.JSON

object HutoolAlias {

    @JvmStatic
    fun main(args: Array<String>) {
        val json = FileUtil.readUtf8String("${Hutool.homeDir}\\class.json")
        JSON.parseObject(json).forEach { k, _ ->
            Hutool.main("$k#alias -d".split(" ").toTypedArray()).also { println() }
        }
    }
}
